src/modules/auth/auth.controller.ts

```diff
@@ -105,4 +119,19 @@ export class AuthController extends Controller {
 
     return toResponse({ data: "Email successfully verified!" });
   }
-}
+
+  @Delete("/signoutAll")
+  @Middlewares(validateAuthentication)
+  public async signout(@Request() req: ExReq): Promise<APIResponse<string>> {
+      const user = getReqUser(req)
+      //refresh token version changed
```

Comment:
the following code is self explanatory, so no need for the comment

Reply:
ok